Emby.Server.Implementations/Library/MediaSourceManager.cs

```diff
@@ -128,6 +128,30 @@ private List<MediaStream> GetMediaStreamsForItem(List<MediaStream> streams)
             return streams;
         }
 
+        /// <inheritdoc />
+        public List<MediaAttachment> GetMediaAttachments(MediaAttachmentQuery query)
+        {
+            return _itemRepo.GetMediaAttachments(query);
+        }
+
+        /// <inheritdoc />
+        public List<MediaAttachment> GetMediaAttachments(string mediaSourceId)
```

Comment:
IMO we should stop passing around GUIDs as strings, and deprecate functions that accept the Id as a string.
i.e. remove this function